### PR TITLE
feat(install): Add operator yaml for 1.4.0 in AWS

### PR DESCRIPTION
**Pull request by ranjithwingrider**
- Add operator YAML for making availability of  1.4 in AWS market place

Signed-off-by: Ranjith R <ranjith.raveendran@mayadata.io>

### CODE REVIEW

File: docs/openebs-operator-1.4.0-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: OPENEBS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "true"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-operator"
```
Comment (ChandanSagar) [inline docs/openebs-operator-1.4.0-aws.yaml]: Can we change the installer type to openebs-aws-operator.

File: docs/openebs-operator-1.4.0-aws.yaml [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+        - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/m-exporter:1.4.0-latest"
+        - name: OPENEBS_IO_CSTOR_POOL_EXPORTER_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/m-exporter:1.4.0-latest"
+        # OPENEBS_IO_ENABLE_ANALYTICS if set to true sends anonymous usage
+        # events to Google Analytics
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "true"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-operator"
```
Comment (ChandanSagar) [inline docs/openebs-operator-1.4.0-aws.yaml]: Same here too, Please change the installer type to openebs-aws-operator.

File: docs/openebs-operator-1.4.0-aws.yaml [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+            initialDelaySeconds: 30
+            periodSeconds: 60
+        # OPENEBS_MAYA_SERVICE_NAME provides the maya-apiserver K8s service name,
+        # that snapshot controller should forward the snapshot create/delete requests.
+        # If not present, "maya-apiserver-service" will be used for lookup.
+        # This is supported for openebs provisioner version 0.5.3-RC1 onwards
+        #- name: OPENEBS_MAYA_SERVICE_NAME
+        #  value: "maya-apiserver-apiservice"
+        - name: snapshot-provisioner
+          image: quay.io/openebs/snapshot-provisioner:1.4.0
```
Comment (muratkars) [inline docs/openebs-operator-1.4.0-aws.yaml]:
```suggestion
          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-315009537/quay.io/openebs/snapshot-provisioner:1.4.0-latest
```

File: docs/openebs-operator-1.4.0-aws.yaml [hunk 4]
```diff
@@ -0,0 +1,682 @@
...
+    metadata:
+      labels:
+        name: openebs-snapshot-operator
+        openebs.io/component-name: openebs-snapshot-operator
+        openebs.io/version: 1.4.0
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+        - name: snapshot-controller
+          image: quay.io/openebs/snapshot-controller:1.4.0
```
Comment (muratkars) [inline docs/openebs-operator-1.4.0-aws.yaml]:
```suggestion
          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-315009537/quay.io/openebs/snapshot-controller:1.4.0-latest
```

**Pull request merged**